pkg/comp-functions/functions/vshnpostgres/maintenance.go

```diff
@@ -116,6 +127,9 @@ func addSchedules(ctx context.Context, svc *runtime.ServiceRuntime) *xfnproto.Re
 		return runtime.NewFatalResult(fmt.Errorf("cannot get cluster object: %w", err))
 	}
 
+	extraEnvVars[3].Value = strconv.FormatBool(comp.Spec.Parameters.Service.RepackEnabled)
```

Comment:
You could use `append()` and assign the array to a new variable to add new envVars instead of manipulating some hard coded array indices.

```suggestion
	additionalVars := append(extraEnvVars, corev1.EnvVar(..))

  // line 145
  WithExtraEnvs(additionalVars...)
```

That will make it more readable and easiert to maintain.

Reply:
implemented and tested it, looks fine